sphinxcontrib/confluencebuilder/__init__.py

```diff
@@ -14,7 +14,7 @@
 from sphinx.writers.text import STDINDENT
 import argparse
 
-__version__='0.9.0.dev0'
+__version__='0.9.0.dev1'
```

Comment:
The version value is still in this PR.

Reply:
Not sure where this is coming from.  My latest changes fixed this and 'view changes' for my second round of markups confirms this.

Reply:
When I examine the change outside GitHub, the `__version__` change is easier to see (I find that there's some oddities when using GitHub's PR/review capabilities). Since this is queued for next release cycle, I'm no longer worried about this (i.e. version values will re-sync when going back into `master`).